backend/api/tests/test_admin.py

```
@@ -376,4 +376,11 @@ def test_export_video_data(self):
         # Verify data row
         self.assertEqual(rows[1][0], str(self.video.id))
         self.assertEqual(rows[1][1], self.video.title)
-        self.assertEqual(rows[1][2], self.video.uploader.email)
\ No newline at end of file
+        self.assertEqual(rows[1][2], self.video.uploader.email)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)